tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SparqlViewsSpec.scala

```diff
@@ -185,15 +185,15 @@ class SparqlViewsSpec extends BaseSpec with EitherValuable with CirceEq {
       }
     }
 
-    "fetch statistics for defaultSparqlIndex" ignore eventually {
+    "fetch statistics for defaultSparqlIndex" in eventually {
```

Comment:
There is a failed event in this test

Reply:
Yes, the failed one is related to the bug I am fixing now